docs/guides/authentication.md

```diff
@@ -19,7 +19,7 @@ A note on terminology:
 
 Username / Password authentication can be enabled on the Admin UI.
 
-> **Note:** Admin Authentication will only restrict access to the Admin _UI_. To also restrict access to the _API_, you must setup [Access control](/docs/guides/access-control.md) config.
+> ❗ **Note:** Admin Authentication will only restrict access to the Admin _UI_. It _will not_ restrict API access. To also restrict access to the API, you must set up [Access controls](/docs/guides/access-control.md).
```

Comment:
Please avoid using emojis in documentation. The "highlight" styling for `blockquote` will suffice if we use them sparingly.

```suggestion
> **Note:** Admin Authentication will only restrict access to the Admin _UI_. It _will not_ restrict API access. To also restrict access to the API, you must set up [Access controls](/docs/guides/access-control.md).
```

Reply:
Removed the emoji.